attls/Jenkinsfile

```diff
@@ -0,0 +1,88 @@
+def serverCredentials = []
+def ARTIFACTORY_CREDENTIALS_ID = 'zowe.jfrog.io'
+Map TEST_SERVERS = [
+        'marist'  : [
+                ansible_host     : 'marist-1',
+                ssh_hostport     : 'ssh-marist-server-zzow01-hostport',
+                ssh_userpass     : 'ssh-marist-server-zzow01',
+                node_home_pattern: '/ZOWE/node/node-{NODE_VERSION}-os390-s390x'
+        ],
+        'marist-2': [
+                ansible_host: 'marist-2',
+                ssh_hostport: 'ssh-marist-server-zzow02-hostport',
+                ssh_userpass: 'ssh-marist-server-zzow02'
+        ],
+        'marist-3': [
+                ansible_host: 'marist-3',
+                ssh_hostport: 'ssh-marist-server-zzow03-hostport',
+                ssh_userpass: 'ssh-marist-server-zzow03'
+        ]
+];
+TEST_SERVERS.each { key, host ->
+    def hostKey = host['ansible_host'].replaceAll(/[^A-Za-z0-9]/, '_').toUpperCase()
+    serverCredentials.add(usernamePassword(
+            credentialsId: host['ssh_hostport'],
+            passwordVariable: "${hostKey}_SSH_PORT".toString(),
+            usernameVariable: "${hostKey}_SSH_HOST".toString()
+    ))
+    serverCredentials.add(usernamePassword(
+            credentialsId: host['ssh_userpass'],
+            passwordVariable: "${hostKey}_SSH_PASSWORD".toString(),
+            usernameVariable: "${hostKey}_SSH_USER".toString()
+    ))
+}
+pipeline {
+
+    agent {
+        label 'zowe-jenkins-agent'
+    }
+
+    parameters {
+        booleanParam(name: 'RELEASE', defaultValue: false, description: 'Toggle this to publish release')
```

Comment:
Is there a difference between Patch, Minor and Major release?

Reply:
my intention was to increase patch only for now as I dont expect any serious development for this library other than occasional patches

Reply:
Ok, we can for now, but I expect that we will pull in other zOS related code.